code/online_challenges/src/codechef/CHDIGER/README.md

```diff
@@ -0,0 +1,2 @@
+# Problem Link:
+[CHDIGER](https://www.codechef.com/MARCH19B/problems/CHDIGER).
```

Comment:
Add more details than just a link. Look at the READMEs for other coding challenges.